### ISSUE TITLE
http no controls.html file

### ISSUE THREAD

**Issue by leonidukg** [mpc-qt/mpc-qt#121]
When http server is enabled, player controls are not available because there is no controls.html file.

**Comment by cmdrkotori** [comment]
Yeah it took me a year to add the web server.  Simply put I felt like cutting my losses and limited the features to the minimum needed to implement it.

**Comment by ElSasori69** [comment]
> Yeah it took me a year to add the web server due to deteriorating health. Simply put I felt like cutting my losses and limited the features to the minimum needed to implement it.

Can you tell me where are the files located when someone installs the program? All I want to do is add an HTML to see progress of the time. I'm on lubuntu btw.

**Comment by cmdrkotori** [comment]
>Can you tell me where are the files located when someone installs the program? All I want to do is add an HTML to see progress of the time. I'm on lubuntu btw.

The "files" that come with mpc-qt are embedded in the program.

BUT, like mpc-hc, you can point mpc-qt at the directory containing your custom html files, and the "special" files (like command.hmtl, variables.html, info.html et al) will pass through to the inbuilt ones.

![Screenshot_20240509_202819](https://github.com/mpc-qt/mpc-qt/assets/65291121/8dc74d05-97d7-4244-9781-5d76f2023449)

The browse button wasn't implemented in the last version, but the fix for it is now in git master.

Note that the web backend isn't throughly tested because I lost access to the original bugtracker where the feature request was made. (it had several sample files to test.)
